### PR TITLE
moves localstack to testcontainers dev mode

### PR DESCRIPTION
**Pull request by rajadilipkolli**
- Moves LocalstackContainers configuration to dev mode

### CODE REVIEW

File: generators/server/templates/app/src/test/java/TestApplication.java [hunk 1]
```diff
@@ -13,6 +13,11 @@
...
 import org.testcontainers.containers.MariaDBContainer;
 <%_ } _%>
+<%_ if (features.includes('localstack')) { _%>
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+<%_ } _%>
+import org.testcontainers.utility.DockerImageName;
     
 @TestConfiguration(proxyBeanMethods = false)
 public class TestApplication {
```
Comment (sivaprasadreddy) [inline generators/server/templates/app/src/test/java/TestApplication.java:23]: Usually a common approach is having a separate `ContainersConfig.java` and `TestApplication.java`

Comment (rajadilipkolli) [reply generators/server/templates/app/src/test/java/TestApplication.java:23]: I got inspire from start.spring.io generated code where both are present in same class. earlier, I used to manage two classes as you have suggested. As I don't see any significant gain creating two classes, can we continue in a single class?

Comment (sivaprasadreddy) [reply generators/server/templates/app/src/test/java/TestApplication.java:23]: I still see start.spring.io generating 2 different files: https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=17&groupId=com.sivalabs&artifactId=myapp&name=myapp&description=Demo project for Spring Boot&packageName=com.sivalabs.myapp&dependencies=lombok,web,data-jpa,flyway,h2,postgresql,validation,actuator,devtools,configuration-processor,testcontainers

Comment (sivaprasadreddy) [reply generators/server/templates/app/src/test/java/TestApplication.java:23]: Aah, my bad. Yes it's generating a single file.

**Review by sivaprasadreddy — APPROVED** [review]

**Pull request merged**